spec/system/candidate_interface/entering_details/international_candidate_enters_their_other_qualification_spec.rb

```diff
@@ -130,10 +143,19 @@ def when_i_fill_in_my_qualification
     fill_in t('application_form.other_qualification.award_year.label'), with: '2015'
   end
 
+  def when_i_fill_in_my_second_qualification
+    fill_in t('application_form.other_qualification.subject.label'), with: 'Believing in the Heart of the Cards part II'
```

Comment:
Shall we use a completely different subject name here, to avoid any potential false positives from Capybara doing partial matching?